php/themes.php

```diff
@@ -165,6 +165,26 @@ function LoadThemes(&$loggedInUser, &$config){
 	return $themes;
 }
 
+// Returns an array of the JAMS_CONSIDERED_RECENT most recent themes
+function GetRecentThemes() {
+	global $dbConn, $config;
+	$escapedJamsConsideredRecent = mysqli_real_escape_string($dbConn,$config["JAMS_CONSIDERED_RECENT"]["VALUE"]);
+	$sql = "
```

Comment:
By the time this is required when handling actions, both $jams and $themes are already populated with this data, so it shouldn't actually require any additional DB queries, instead it can be checked by only looking at the themes in $jams[$id]["theme"]